backend/src/main/java/gov/cdc/usds/simplereport/service/OktaServiceEmptyImpl.java

```diff
@@ -1,36 +1,54 @@
 package gov.cdc.usds.simplereport.service;
 
-import org.springframework.stereotype.Component;
-import org.springframework.context.annotation.Primary;
-import org.springframework.context.annotation.Profile;
-
-import gov.cdc.usds.simplereport.config.BeanProfiles;
-
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.HashMap;
+
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRoleClaims;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
 import gov.cdc.usds.simplereport.service.model.IdentityAttributes;
 
 /**
  * Created by jeremyzitomer-usds on 1/7/21
  * 
  * Handles all user/organization management in Okta
  */
-@Component
-@Primary
-@Profile(BeanProfiles.NO_OKTA_MGMT)
 public class OktaServiceEmptyImpl implements OktaService {
 
-    public OktaServiceEmptyImpl() {}
+    Map<String,OrganizationRoleClaims> usernameRolesMap;
+    Map<String,Map<OrganizationRole,List<String>>> orgRoleUsernamesMap;
+
+    public OktaServiceEmptyImpl(Map<String,OrganizationRoleClaims> usernameRolesMap) {
+        this.usernameRolesMap = usernameRolesMap;
+        this.orgRoleUsernamesMap = new HashMap<>();
+        for (Map.Entry<String,OrganizationRoleClaims> entry : usernameRolesMap.entrySet()) {
+            Map<OrganizationRole,List<String>> roleUsernamesMap = 
+                    orgRoleUsernamesMap.getOrDefault(entry.getValue().getOrganizationExternalId(),
+                                                     new HashMap<>());
```

Comment:
i am going to gently decline for a) readability and b) because i think this code block doesn't address the case of adding usernames to `roleUsernamesMap` when `roleUsernamesMap` is already non-null; and after we add that back in, what i have now feels a touch more straightforward. considering this is just a demo class, i'm going to leave as is. (though I'll pull out orgId as its own variable, to make things a touch more readable.)